### PR TITLE
Move osde2e wrapper to its own folder and move common functions to a lib

### LINKED ISSUE #46: Add argument to install addons on cluster after it is installed
There are some addons that could be installing using rosa after cluster is installed, for example, cluster-logging-operator

Adding a parameter --addons to the wrapper, we can include a list of addons to be added after installation.

rosa install addon -c [clustername] cluster-logging-operator

### PR DESCRIPTION
**Pull request by morenod**
Before adding the rosa wrapper, this PR will move osde2e-wrapper to its own folder and it will also move common functions used by both script to another file, to avoid duplicated works

### CODE REVIEW

**Comment by morenod** [comment]
HOLD until https://github.com/cloud-bulldozer/perfscale-managed-services/pull/56 and https://github.com/cloud-bulldozer/perfscale-managed-services/pull/50 are merged

**Comment by morenod** [comment]
Needs to be merge at the same time than https://github.com/cloud-bulldozer/scale-ci-pipeline/pull/178

**Review by chentex — APPROVED** [review]
This looks really good

**Review by chentex — APPROVED** [review]
LGTM

File: .github/workflows/tests.yml
Comment (dry923) [inline .github/workflows/tests.yml]: I think the install should probably be rosa/requirements.txt right?

**Review by dry923 — APPROVED** [review]
LGTM! Waiting on signoff of the pipeline job then will merge both at the same time

**Pull request merged**
